packages/blade/src/components/Form/FormLabel.tsx

```diff
@@ -58,11 +58,7 @@ const FormLabel = ({
 
   let necessityLabel: React.ReactNode = null;
 
-  if (!isDesktop && position === 'left') {
```

Comment:
Don't think it would be very useful since this warning is based on responsiveness.. So if you make the window smaller then also would throw error. Which may or may not be what they expect to happen.